_dev/docker-compose.yml

```diff
@@ -20,7 +20,6 @@ services:
     restart: always
   mysql:
     image: mysql:8.0.31
-    platform: linux/amd64
```

Comment:
Have they fixed the image for arm64 arch?

Reply:
When I run the docker-compose with `platform: linux/amd64` it gives the following error
```
_dev git:(feature/skaffold) ✗ docker-compose up
image with reference mysql:8.0.31 was found but does not match the specified platform: wanted linux/amd64, actual: linux/arm64/v8
```

With `platform: linux/arm64/v8`, it works
And without the platform it works as well